packaging/packager_rpm.sh

```diff
@@ -12,30 +12,33 @@ dnf -y install python3 python3-coverage
 dnf -y install python3-pillow
 # pip dependencies (for dependencies not available as RPM)
 dnf -y install gcc libX11-devel libXtst-devel python3-devel libpng-devel python3-pip redhat-rpm-config
+
 pip3 install --upgrade pip
+pip3 install setuptools setuptools-rust
+pip3 install pillow==10.4.0
+
 # contour, template, feature, cascade, text matching
 dnf -y install python3-numpy python3-opencv
+pip3 install numpy opencv-python
```

Comment:
I don't understand the duplications with the dnf at any point here. Why we installing everything twice?